recipe/meta.yaml

```
@@ -10,34 +10,38 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://github.com/zerothi/{{ name }}/archive/v{{ version }}.tar.gz
   sha256: {{ sha256 }}
+  patches:
+    - patches/random_failure_seed.patch
 
 build:
-  number: 1
+  number: 2
   skip: true  # [win]
 
 requirements:
   build:
     - {{ compiler('fortran') }}
     - {{ compiler('c') }}
   host:
+    - setuptools
+    - libblas
     - libcblas
     - liblapack
     - python
-    - setuptools
-    # once we hit 1.16.3 it should be fine again
-    # I read that numpy-devel should work, so...
-    - numpy <=1.15.3|>=1.16.3
-    - scipy >=0.18
-    # For cythonizing stuff
     - cython
-    - pyparsing
+    # this is for the sisl requirement
+    - numpy >=1.13
+    # this is for conda-smithy so that the global
+    # pinning is correctly specified
+    - numpy
+    - scipy >=0.18
     - netcdf4
+    - pyparsing >=1.5.7
   run:
-    - liblapack
     - python
-    - pyparsing
-    - {{ pin_compatible('numpy') }}
-    - {{ pin_compatible('scipy') }}
+    - pyparsing >=1.5.7
+    - {{ pin_compatible('numpy', min_pin='1.13') }}
```

This should not include an explicit version number. The build won't be compatible with `1.13`, it will only be compatible with the `numpy` version in `host` and newer ones, ie. `>=1.17` / `>=1.19`  depending on the Python version.
```suggestion
    - {{ pin_compatible('numpy') }}
```